src/pages/api/ms_notes.ts

```
@@ -1,7 +1,7 @@
 import { getCollection } from "astro:content";
-import { featureFlags } from "@config/featureFlag/featureFlag.json";
+import featureFlagConfig from "@config/featureFlag/featureFlag.json";
 
-export async function GET({ request }: { request: Request }) {
+export async function GET({  }: { request: Request }) {
```

Biome (2.1.2)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)